infra/modules/providers/azure/cosmosdb/output.tf

```diff
@@ -0,0 +1,10 @@
+output "cosmosdb_endpoint" {
+  description = "The endpoint used to connect to the CosmosDB account."
+  value       = "${azurerm_cosmosdb_account.cosmosdb.endpoint}"
+}
+
+output "cosmosdb_primary_master_key" {
```

Comment:
You may want to consider adding the `connection_strings` here as well?

Reply:
Will do, pushing that change in the next 20 minutes